### PR TITLE
#48 Angular10 Update

### PR DESCRIPTION
**Pull request by VictorDeBlas**

### CODE REVIEW

**Review by alfonsserra — APPROVED** [review]

File: projects/systelab-charts/package.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
     "url": "https://github.com/systelab/systelab-charts.git"
   },
   "private": false,
+  "dependencies": {
+    "tslib": "^2.0.0"
+  },
   "peerDependencies": {
     "@angular/common": ">=9.0.0 <10.0.0",
```
Comment (vinyulis) [inline projects/systelab-charts/package.json:20]: Why these dependencies are not changed? Should be >=10 <11?

File: projects/systelab-charts/package.json [hunk 2]
```diff
@@ -13,6 +13,9 @@
     "url": "https://github.com/systelab/systelab-charts.git"
```
Comment (vinyulis) [inline projects/systelab-charts/package.json:13]: Could we increase the major version of this package.json?

**Review by vinyulis — CHANGES_REQUESTED** [review]

File: projects/systelab-charts/package.json (hunk 2 above)
Comment (VictorDeBlas) [reply projects/systelab-charts/package.json:13]: Done

**Review by vinyulis — APPROVED** [review]
OK!!. Thanks!!.

**Review by carlosra85 — APPROVED** [review]

**Pull request merged**